pkg/agent/openflow/pipeline.go

```diff
@@ -254,6 +255,8 @@ var (
 	// endpointIPRegRange takes a 32-bit range of register endpointIPReg to store
 	// the selected Service Endpoint IP.
 	endpointIPRegRange = binding.Range{0, 31}
+	// the selected Service Endpoint IPv6.
+	endpointIPv6XXRegRange = binding.Range{0, 127}
```

Comment:
We should keep the same convention of writing XXReg in the whole patch (some code use XxReg)

Reply:
OK